src/request/request.ts

```diff
@@ -34,7 +34,7 @@ export class Request implements IRequest {
 
   public getOrFail(key: string): any {
     const value = this.get(key)
-    if (value) {
+    if (value !== undefined) {
```

Comment:
because the key does exist and it contains a falsely value (which is information). failing would imply that they key doesn't exist when in fact it might. 

there could be debugging case where someone has a key but when they try to 'get' they see it's a null value so they figure they may have an issue with their code not properly passing a value to that key. else if we leave it as it is, they might think the key is not there at all when it really is.

Reply:
maybe we should be checking `typeof value !== "undefined"`
@moraneva